frame/support/src/lib.rs

```diff
@@ -1858,21 +1859,6 @@ pub mod pallet_prelude {
 /// pub(super) type MyStorage<T> = StorageValue<Value = u32>;
 /// ```
 ///
-/// The optional attribute `#[pallet::whitelist_storage]` will declare the
```

Comment:
We should still list the possible attributes as some list linking to the macros

Reply:
Done. I ended up keeping excerpts of the original docs in the long docs page on `pallet::pallet` and then link out to full versions of these containing expansion info and some other stuff on the individual macro stubs